brave/browser-laptop
# Ads Test Fixes (Issue #61/Sections 1 - 3)

Addressing Sections 1 - 3 of internal issue #61

## Submitter Checklist:


## Test Plan:
1. Confirm that Brave can be packaged and installed via the Developer channel with the name `Brave-Ads-Test`
2. Confirm that the typographical issues in Section 1 of #61 have been addressed.
3. Confirm that changing the place via the "Place" dropdown in `about:preferences#ads` logs a setting change event as expected.
4. Confirm with several profile installs that the "Garbled text issue" no longer occurs when first opting in to Brave ads. (See #61 Section 3 Issue 1)

## Reviewer Checklist:


Tests
